src/Turbo/src/Helper/TurboStream.php

```diff
@@ -61,15 +61,15 @@ public static function remove(string $target): string
      */
     public static function before(string $target, string $html): string
     {
-        return self::wrap('before', $target, $html);
+        return self::custom('before', $target, $html);
     }
 
     /**
      * Inserts after the element(s) designated by the target CSS selector.
      */
     public static function after(string $target, string $html): string
     {
-        return self::wrap('after', $target, $html);
+        return self::custom('after', $target, $html);
```

Comment:
I think we should revert these changes (other ones too in existing methods)

(wrap worked better as method name here)